### ISSUE TITLE
install fails with lastest stable on linuxbrew

### ISSUE THREAD

**Issue by overratedpro** [dcos/dcos-e2e#967]
Using linuxbrew (https://github.com/Linuxbrew/brew.git) version 1.6.6 installation:
```bash
brew install https://raw.githubusercontent.com/dcos/dcos-e2e/master/dcosdocker.rb
```

Expected result:
brew installs the dcos-docker binary

Actual result:
```
==> /home/linuxbrew/.linuxbrew/Cellar/dcosdocker/2018.06.11.1/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /tmp/dcosdocker--cffi-20180612-20601-26zr9r/cffi-1.11.5
Last 15 lines from /home/linuxbrew/.cache/Homebrew/Logs/dcosdocker/17.pip:
  Removing source in /tmp/pip-req-build-b_yo337f
Command "/home/linuxbrew/.linuxbrew/Cellar/dcosdocker/2018.06.11.1/libexec/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-req-build-b_yo337f/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-xd7ko9wq/install-record.txt --single-version-externally-managed --compile --install-headers /home/linuxbrew/.linuxbrew/Cellar/dcosdocker/2018.06.11.1/libexec/include/site/python3.6/cffi" failed with error code 1 in /tmp/pip-req-build-b_yo337f/
Exception information:
Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/Cellar/dcosdocker/2018.06.11.1/libexec/lib/python3.6/site-packages/pip/_internal/basecommand.py", line 228, in main
    status = self.run(options, args)
  File "/home/linuxbrew/.linuxbrew/Cellar/dcosdocker/2018.06.11.1/libexec/lib/python3.6/site-packages/pip/_internal/commands/install.py", line 335, in run
    use_user_site=options.use_user_site,
  File "/home/linuxbrew/.linuxbrew/Cellar/dcosdocker/2018.06.11.1/libexec/lib/python3.6/site-packages/pip/_internal/req/__init__.py", line 49, in install_given_reqs
    **kwargs
  File "/home/linuxbrew/.linuxbrew/Cellar/dcosdocker/2018.06.11.1/libexec/lib/python3.6/site-packages/pip/_internal/req/req_install.py", line 779, in install
    spinner=spinner,
  File "/home/linuxbrew/.linuxbrew/Cellar/dcosdocker/2018.06.11.1/libexec/lib/python3.6/site-packages/pip/_internal/utils/misc.py", line 698, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip._internal.exceptions.InstallationError: Command "/home/linuxbrew/.linuxbrew/Cellar/dcosdocker/2018.06.11.1/libexec/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-req-build-b_yo337f/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-xd7ko9wq/install-record.txt --single-version-externally-managed --compile --install-headers /home/linuxbrew/.linuxbrew/Cellar/dcosdocker/2018.06.11.1/libexec/include/site/python3.6/cffi" failed with error code 1 in /tmp/pip-req-build-b_yo337f/

Do not report this issue to Homebrew/brew or Homebrew/core!

```

**Comment by overratedpro** [comment]
Thank you, @adamtheturtle for the research done with the version 1.6.8. 
My OS is Ubuntu 17.10 server edition running stock kernel (4.13.0-43-generic #48-Ubuntu SMP Wed May 16 12:18:48 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux).
Please note this is not a container not a VM.
I'll try to upgrade the Linuxbrew version to 1.6.8 and see if the problem is repoducible there.

**Comment by adamtheturtle** [comment]
Thank you - @overratedpro please can you try using the `--verbose` flag with Linuxbrew to see if there is more information given.

**Comment by adamtheturtle** [comment]
@overratedpro I have reproduced the issue by following the same instructions that I posted above but with `vagrant init ubuntu/xenial64; vagrant up --provider virtualbox`. I will find and fix the issue as soon as possible.

**Comment by adamtheturtle** [comment]
I believe that the fix is to require `pkg-config`. I will test this now and ship a fix today if that is the issue.

**Comment by adamtheturtle** [comment]
I have released a new version of DC/OS E2E, version `2018.06.14.1`.
I believe that this should fix your issue.
Please re-open if the issue persists, or you hit any other issues.

Thank you for the report!
